### PR TITLE
Adds a Jim Norton's Quebecois Coffee shop to more stations

### PR DESCRIPTION
**Pull request by nyozzl**
## About The Pull Request
Jim nortons!

Adds a jim nortons coffeeshop to delta and icebox

## Why It's Good For The Game
Finally people on delta, icebox, and birdshot can now enjoy a signature Jim Norton's coffee

Coffee shops are cool 👍 

🆑
map: Jim Norton's Quebecois Coffee has expanded its business to other stations
/:cl:

Delta:
image

Icebox:
image

### CODE REVIEW

**Comment by distributivgesetz** [comment]
yooo ive been saying this

**Comment by Singul0** [comment]
rip maint size

**Comment by AtomTheProphet** [comment]
That is definitely a great place to put it in Delta. Good choice.

File: _maps/map_files/Birdshot/birdshot.dmm [hunk 1]
```diff
@@ -19907,6 +19976,19 @@
...
 /area/station/commons/fitness/locker_room)
+"hcr" = (
+/obj/effect/turf_decal/trimline/green/line,
+/obj/item/clothing/suit/apron/chef{
+	name = "Jim Norton's Quebecois Coffee apron"
+	},
+/obj/item/clothing/head/soft/green{
+	color = "#00a62e";
+	desc = "It's a baseball hat in a tastefull green colour.";
+	name = "Jim Norton's Quebecois Coffee cap"
```
[the remark this answers is not in the document]
Comment (Pepsilawn) [reply _maps/map_files/Birdshot/birdshot.dmm:19987]: Same for all the other maps'

**Comment by EuSouAFazer** [comment]
The "why it's good for the game" is mandatory, specially when it's uneeded sterilization/mapping standardization. Changes like this need good reasons to be implemented; PRs like this make maps less special by reducing how many special and unique stuff are in each map by making them no longer special or unique

**Comment by Loafin34** [comment]
I think nyozzl wants to be able to do the Jim Norton's gimmick on every map, which is fair. It's like the commissary except more specific.

**Comment by LemonInTheDark** [comment]
Soul death

File: _maps/map_files/Birdshot/birdshot.dmm (hunk 1 above)
[the remark this answers is not in the document]
Comment (nyozzl) [reply _maps/map_files/Birdshot/birdshot.dmm:19987]: gonna be honest these are just the hat and apron from meta station jim nortons just a renamed apron is the improper needed

**Comment by nyozzl** [comment]
/datum/unit_test/log_mapping
Error: FAIL /datum/unit_test/log_mapping 0s

guh?

**Comment by tttruancy** [comment]
Removes the peak ghetto botany/chef botany spot from icebox for coffeeshop.. sorry but the maint shops I've made through that wall are better for the game than barista RP

**Comment by LemonInTheDark** [comment]
> /datum/unit_test/log_mapping Error: FAIL /datum/unit_test/log_mapping 0s
> 
> guh?

for clarity

image

**Pull request closed without merging**